modules/income_and_assets/lib/income_and_assets/helpers.rb

```diff
@@ -71,6 +71,30 @@ def split_currency_amount_lg(amount, field_lengths = {})
       amount_hash
     end
 
+    # NOTE: in regards to the checkbox_value and radio_yesno helpers below,
+    #   HexaPDF is more strict about the values it accepts for checkboxes and radio buttons
+    #   than PDFtk. HexaPDF wants true/false for checkboxes and strings/symbols for radio buttons.
+
+    ##
+    # Converts a value to a checkbox-compatible boolean.
+    #
+    # @param value [Any]
+    # @return [Boolean]
+    #
+    def checkbox_value(value)
+      value ? true : false
```

Comment:
I couldn't find a way that the linter would accept to cast the value to a Boolean, so this will have to suffice